### ISSUE TITLE
About hs.chooser:rows

### ISSUE THREAD

**Issue by tomtrije** [Hammerspoon/hammerspoon#1725]
When i received changes of query, i want to change choises, and i did it. However, i can not change the height by **hs.chooser:rows** until invoke hs.chooser:show again. I like the height is auto with the content, but i can't implement, pls help me, thanks.

**Comment by latenitefilms** [comment]
I think I get around this by hiding and showing the `hs.chooser`. It happens so fast the user doesn't notice.

**Comment by tomtrije** [comment]
Yes, just like you said, i can upgrade by show() method, but it happens so fast around by hiding and showing.
**I hopes that there is a method can upgrade rows automatically, just like Afred.** And i can set a max rows when the hs.chooser inited. 

Best wish for hammerspoon, it's my love.

**Comment by latenitefilms** [comment]
I'm not sure if this is helpful or not, but this is how we handle things:

https://github.com/CommandPost/CommandPost/blob/develop/src/plugins/core/action/manager/activator.lua
